### PR TITLE
Add move 2024 Conventions as comments to the new package example

### PR DESCRIPTION
**Pull request by yanganto**
## Description 
* Fix typo in 2024 Conventions
* ~Add Conventions as comments to a new package when `sui move new`~
* Add Conventions link to a new package when `sui move new`

## Test plan 
* no extra tests needed

## Release notes
- [X] CLI: Add Conventions as comments to a new package when `sui move new`

### CODE REVIEW

**Comment by damirka** [comment]
Thank you for your PR @yanganto! It is a fresh and interesting improvement, but I'm afraid filling every new project wit coding conventions may come out as noisy, and instead I suggest changing it to a single line with a link to coding conventions. Let me know if you're comfortable doing that, or we can pick it up some other time.

**Comment by yanganto** [comment]
For me, I will not insist on this or anything.

For developer experience, they just want to deliver a product with high quality, and do not want anyone to say their code is bad after product shipped.

Previous commit is exactly a new developer will want in my imagine.
"Just tell them what is the best practice clearly without asking a lot of document reading and still ambiguous".

File: crates/sui-move/src/new.rs [hunk 1]
```diff
@@ -34,7 +34,12 @@ impl New {
             r#"/*
 /// Module: {name}
 module {name}::{name};
-*/"#,
+*/
+
+// The following recommendations are based on 2024 Move.
```
Comment (damirka) [inline crates/sui-move/src/new.rs:39]: I would suggest changing the text to something like "For naming coding conventions for Move, see <link>"

File: crates/sui-move/src/new.rs [hunk 2]
```diff
@@ -34,7 +34,12 @@ impl New {
             r#"/*
 /// Module: {name}
 module {name}::{name};
-*/"#,
+*/
+
+// For coding conventions for Move, see
```
Comment (tnowacki) [inline crates/sui-move/src/new.rs:39]:
```suggestion
// For Move coding conventions, see
```

**Pull request merged**